### PR TITLE
react webpack app

### PR DESCRIPTION
**Pull request by jaamanu**
In this PR 
Create greeting api app

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @jaamanu,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] You didn't document your work professionally:
   - You forgot to add your README file. Please use the [Microverse README template](https://github.com/microverseinc/readme-template) and edit it with your personal and project data.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: README.md
Comment (brytebee) [inline README.md:1]: Kindly restore this file.

**Review by brytebee — COMMENTED** [review]
Hi, @jaamanu 👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



You have submitted a project, that is missing the README file. See inline comments below.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @jaamanu,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] You didn't document your work professionally:
   - You forgot to add your README file. Please use the [Microverse README template](https://github.com/microverseinc/readme-template) and edit it with your personal and project data.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by rotimiazeez — APPROVED** [review]
Hi @jaamanu  👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job implementing the project requirements 👏🏾 
- Good and Descriptive README 👍🏼
- No linter errors ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**